plugins/polio/js/src/domains/Campaigns/MainDialog/CreateEditDialog.tsx

```diff
@@ -283,7 +213,7 @@ const CreateEditDialog = ({ isOpen, onClose, campaignId }) => {
                     {formatMessage(MESSAGES.cancel)}
                 </Button>
                 <Button
-                    onClick={formik.handleSubmit}
+                    onClick={() => formik.handleSubmit()}
```

Comment:
why ?

Reply:
There's a weird bug otherwise. Don't remember what exactly, but it's necessary